### PR TITLE
Merge master into temp/docs

### PR DESCRIPTION
**Pull request by najeeb-kazmi**
Towards #314 

Update docs to 1.5.0.

### CODE REVIEW

File: src/python/nimbusml.pyproj [hunk 1]
```diff
@@ -30,69 +30,70 @@
     <Compile Include="docs\sphinx\conf.py" />
     <Compile Include="nimbusml\base_predictor.py" />
     <Compile Include="nimbusml\base_transform.py" />
-    <Compile Include="nimbusml\cluster\_kmeansplusplus.py" />
+    <Compile Include="nimbusml\cluster\kmeansplusplus.py" />
```
Comment (pieths) [inline src/python/nimbusml.pyproj]: Why were these renamed (see all changes in the file)? `kmeansplusplus.py` (and all the other files) without the underscore should not exist.

File: src/python/nimbusml.pyproj [hunk 2]
```diff
@@ -1199,25 +1231,25 @@
...
-    <Content Include="nimbusml\datasets\_data\test_fs.csv" />
-    <Content Include="nimbusml\datasets\_data\timeseries.csv" />
-    <Content Include="nimbusml\datasets\_data\topics.csv" />
-    <Content Include="nimbusml\datasets\_data\train-250.wikipedia.sample.tsv" />
-    <Content Include="nimbusml\datasets\_data\train-500.uciadult.sample.csv" />
-    <Content Include="nimbusml\datasets\_data\train-msltr.sample.csv" />
-    <Content Include="nimbusml\datasets\_data\train-ticketchoice.csv" />
-    <Content Include="nimbusml\datasets\_data\train-twitter.gen-sample.tsv" />
-    <Content Include="nimbusml\datasets\_data\train_fs.csv" />
+    <Content Include="nimbusml\datasets\data\gplv2\airquality.csv" />
```
Comment (pieths) [inline src/python/nimbusml.pyproj]: Why was this renamed?

**Comment by pieths** [comment]
The CI run (unit tests) are not passing.

File: src/python/nimbusml.pyproj (hunk 2 above)
Comment (najeeb-kazmi) [reply src/python/nimbusml.pyproj]: Good catch. I must have missed this because there were no conflicts in nimbusml.pyproj. This is also why the CI run fails (`TestPyproj.test_pyproj_existence` and `TestPyproj.test_pyproj_in` - unit tests are passing.
